### PR TITLE
Update workflows

### PR DESCRIPTION
**Pull request by wardbeyens**

### CODE REVIEW

File: .github/workflows/firebase-hosting-merge-beta.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Install Dependencies
+        run: yarn
+      - name: Build project
+        run: yarn run build:prod
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          firebaseServiceAccount: "${{ secrets.FIREBASE_SERVICE_ACCOUNT_WTMG_PRODUCTION }}"
+          channelId: live
+          projectId: wtmg-production
```
Comment (wardbeyens) [inline .github/workflows/firebase-hosting-merge-beta.yml:34]: this

Comment (th0rgall) [reply .github/workflows/firebase-hosting-merge-beta.yml:34]: We should add a target, indeed:
- GH action docs: https://github.com/FirebaseExtended/action-hosting-deploy#target-string
- Firebase docs: https://firebase.google.com/docs/hosting/multisites?authuser=0&hl=en#set_up_deploy_targets

I think we should proceed as follows:
1. Follow the Firebase docs to apply a target name to the prod and beta sites (`firebase target:apply hosting ... ...` - then the target name should be registered.
2. Use those target names in the actions according to the GH action docs.

Comment (wardbeyens) [reply .github/workflows/firebase-hosting-merge-beta.yml:34]: Yes I like this idea

**Review by th0rgall — COMMENTED** [review]
I added the last env variables. Let's see how this runs!

**Pull request merged**